test/integration/test_language_translator_v3.py

```diff
@@ -5,39 +5,42 @@
 import pytest
 import os
 
-@pytest.mark.skipif(
-    os.getenv('VCAP_SERVICES') is None, reason='requires VCAP_SERVICES')
+
+@pytest.mark.skipif(os.getenv('VCAP_SERVICES') is None,
+                    reason='requires VCAP_SERVICES')
 class TestIntegrationLanguageTranslatorV3(unittest.TestCase):
+
     @classmethod
     def setup_class(cls):
         cls.language_translator = ibm_watson.LanguageTranslatorV3('2018-05-01')
-        cls.language_translator.set_default_headers({
-            'X-Watson-Test':
-            '1'
-        })
+        cls.language_translator.set_default_headers({'X-Watson-Test': '1'})
 
     def test_translate(self):
-        translation = self.language_translator.translate(text='Hello', model_id='en-es').get_result()
+        translation = self.language_translator.translate(
+            text='Hello', model_id='en-es').get_result()
         assert translation is not None
 
     def test_document_translation(self):
-        with open(join(dirname(__file__), '../../resources/hello_world.txt'), 'r') as fileinfo:
+        with open(join(dirname(__file__), '../../resources/hello_world.txt'),
+                  'r') as fileinfo:
             translation = self.language_translator.translate_document(
-                file=fileinfo,
-                file_content_type='text/plain',
+                file=fileinfo, file_content_type='text/plain',
                 model_id='en-es').get_result()
         document_id = translation.get('document_id')
         assert document_id is not None
 
-        document_status = self.language_translator.get_document_status(document_id).get_result()
+        document_status = self.language_translator.get_document_status(
+            document_id).get_result()
         assert document_status is not None
 
         if document_status.get('status') == 'available':
-            response = self.language_translator.get_translated_document(document_id, 'text/plain').get_result()
+            response = self.language_translator.get_translated_document(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many positional arguments for method call](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4994724)